cli/version/version_match.go

```diff
@@ -0,0 +1,174 @@
+package version
+
+import (
+	"fmt"
+
+	"github.com/tarantool/tt/cli/util"
+)
+
+type requiredField uint8
+type requiredFields []requiredField
+
+const (
+	// Values according order of Fields in regex from `matchVersionParts()`.
+	requiredFieldBuildName requiredField = iota
+	requiredFieldMajor
+	requiredFieldMinor
+	requiredFieldPatch
+	requiredFieldReleaseType
+	requiredFieldReleaseNum
+	requiredFieldAdditional
+	requiredFieldHash
+	requiredFieldRevision
+
+	countOfRequiredFields
+)
+
+// NotFoundError is sentinel error for case when required version not found.
+type NotFoundError struct {
+	expected string
+}
+
```

Comment:
Please, add a comment to the public API method.

Reply:
Added the comment for `Error()` method